Week3/homework/js-exercises/dogYears.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+function calculateDogAge(dogAge) {
+    return "Your doggie is " + (dogAge * 7) + " years old in dog years!";
```

Comment:
Ah. I think there is a typo in the homework currently. 7 human years should be equal to 1 dog year instead of other way round. Otherwise, I'm currently 29 years old and my dog will be over 200 years. 😄 

But your logic is correct. However, I would suggest you look into template literals introduced in ES6(https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Template_literals) instead of using double quotes and concatenating strings using + operator.

Reply:
Ah I see.  It seems that I misunderstood the homework, sorry about that.. Also I know what you advice me but it is hard to find in my keyboard that little thing( ` ). I have to search for it and copy paste every time so I did not use it so much.. Anyway I will use it👩‍💻